examples/Authentication/google_ads_php.ini

```diff
@@ -23,6 +23,15 @@ developerToken = "INSERT_DEVELOPER_TOKEN_HERE"
 ; Required OAuth2 credentials. Uncomment and fill in the values for the
 ; appropriate flow based on your use case.
 
+; Application Default Credentials (ADC)
+; If no explicit credentials (clientId/secret/refreshToken or jsonKeyFilePath)
+; are provided below, the client library will automatically attempt to use ADC
+; to find credentials based on the environment (e.g., GOOGLE_APPLICATION_CREDENTIALS
+; environment variable, 'gcloud auth application-default login', or compute engine service account).
+; Explicitly set credentials take precedence over ADC.
+; See https://cloud.google.com/docs/authentication/application-default-credentials for details.
+; The 'useApplicationDefaultCredentials' flag has been removed, as ADC is now the default fallback.
```

Comment:
Did we ever release a version of the client library with the `useApplicationDefaultCredentials` flag?

If we didn't, we can just remove this line.